graphein/protein/tensor/io.py

```diff
@@ -254,6 +261,10 @@ def protein_to_pyg(
     )
     if store_het:
         out.hetatms = [het_coords]
+
+    if store_bfactor:
+        out.bfactor = torch.tensor(df["b_factor"].values)

```

Comment:
I think `torch.from_numpy` might be a little better

Reply:
done